### PR TITLE
Swapper router add callee tests

### PR DESCRIPTION
**Pull request by telome**

### CODE REVIEW

File: test/funnels/callees/UniV3SwapperCallee.t.sol [hunk 1]
```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: UNLICENSED
```
Comment (sunbreak1211) [inline test/funnels/callees/UniV3SwapperCallee.t.sol]:
```suggestion
// SPDX-License-Identifier: AGPL-3.0-or-later
```

**Review by sunbreak1211 — APPROVED** [review]
Excepting for the minor license issue, good for merging.

**Review by oldchili — APPROVED** [review]
LGTM

**Comment by oldchili** [comment]
I'm going to merge this and start merging the other branches as well.

**Pull request merged**